src/server/api/organization.js

```diff
@@ -201,7 +202,7 @@ export const resolvers = {
         messageHandlers,
         actionHandlers,
         unsetFeatures,
-        featuresJSON: JSON.stringify(visibleFeatures)
+        featuresJSON: JSON.stringify(features)
```

Comment:
I'm not certain if there are implications to sending all features vs only "visible".